README.md

```diff
@@ -0,0 +1,204 @@
+
+### Prerequisites
+
+In [Gimbal Manager > Apps](https://manager.gimbal.com/apps) create an app for each iOS and Android.
+(For the purposes of differentiating between them in analytics, it helps to make the names and bundle/package IDs unique.)
+You can find your apps' API keys in the app detail view -- you will need these later.
+
+### Add the plugin
+
+To add the plugin to your app:
+
+```shell
+    cordova plugin add cordova-plugin-gimbal-airship-adapter
+```
+
+## Initialization
+
+This plugin allows configuration and setup via method calls, or via Cordova preferences in your app's `config.xml`.
+
+### Start with preferences
+
+If you want to just get it started, this is for you.
+
+```xml
+<widget ...>
+    ...
+    <preference name="com.gimbal.api_key.android" value="<YOUR_GIMBAL_ANDROID_API_KEY>" />
+    <preference name="com.gimbal.api_key.ios" value="<YOUR_GIMBAL_IOS_API_KEY>" />
+    <preference name="com.gimbal.auto_start" value="true" />
+</widget>
+```
+
+If auto-start is enabled, and the API key is set, Gimbal will start.
+Without an API key, auto-start has no effect.
+
+By default, Airthip Custom Event tracking is enabled for Gimbal Place Entry and Departure events.
+Airship Region Event tracking is disabled by default.
+
+### Start with code
+
+To start the adapter call:
+
+```javascript
+Gimbal.start('<YOUR_GIMBAL_API_KEY>',
+    (started) => console.log('Running Gimbal Airship Adapter: ' + started),
+    () => console.log('Failed to start Gimbal Airship Adapter'));
+```
+
+NOTE: your iOS and Android apps are managed separately in Gimbal Manager, and have separate API keys.
+Your code will need to determine which platform it is running on, so that it may supply the correct API key, e.g. with `cordova-plugin-device`:
+
+```javascript
+var apiKey = null;
+
+switch (device.platform) {
+    case 'iOS':
+        apiKey = '<YOUR_GIMBAL_IOS_API_KEY>';
+        break;
+    case 'Android':
+        apiKey = '<YOUR_GIMBAL_ANDROID_API_KEY>';
+        break;
+    default:
+        console.log('Platform ' + device.platform + ' not supported by Gimbal SDK');
+        return;
+}
+
+Gimbal.start(apiKey,
+    (started) => console.log('Running Gimbal Airship Adapter: ' + started),
+    () => console.log('Failed to start Gimbal Airship Adapter'));
+```
+
+Once the adapter is started, it will automatically resume its last state when the app is restarted, including if started in the background. 
+The API key and the started status are persisted between app starts -- you only need to call `start`  once.
+
+Typically this will be called when the user has opted-in to a feature that benefits from location-triggered Airship notifications, after the appropriate permissions are granted by the user.
+It is also possible to call `start` every time in your `onDeviceReady` callback, but note that the first time that permissions are granted (*after* Gimbal is started), Gimbal will not request location updates from the OS until the next app restart.
```

Comment:
Is this segment true? On iOS, I've been able to get a location-triggered push a few seconds after granting permissions.

Reply:
Adding weasel words!